x-pack/metricbeat/module/aws/billing/_meta/fields.yml

```diff
@@ -0,0 +1,6 @@
+- name: billing
+  type: group
+  description: >
+    `billing` contains the estimated charges for your AWS account in Cloudwatch.
+  release: beta
+  fields:
```

Comment:
Is this supposed to be empty?

Reply:
Yeah I believe so. Because this metricset is created using light module with name `aws.metrics.EstimatedCharges.max` so it's not gonna be `aws.billing.*`.